docs/development/setup-dapr-development-env.md

```diff
@@ -1,57 +1,91 @@
 # Setup Dapr development environment
 
-This document helps you get started developing Dapr. If you find any problem while following this guide, please create a Pull Request to update this document.
+This document helps you get started developing Dapr. If you find any problems while following this guide, please create a Pull Request to update this document.
 
 ## Docker environment
 
 1. Install [Docker](https://docs.docker.com/install/)
-    > For Linux, you'll have to configure docker to run without sudo for this to work, because of the environment variables.  See the following on how to configure [this](https://docs.docker.com/install/linux/linux-postinstall/).
-
-2. Create your [Docker Hub account](https://hub.docker.com)
+    > For Linux, you'll have to configure docker to run without `sudo` for the dapr build scripts to work. Follow the instructions to [manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
 
+2. Create your [Docker Hub account](https://hub.docker.com/signup) if you don't already have one.
 
 ## Golang dev environment
 
 ### Linux and MacOS
 
-1. Go 1.16 [(instructions)](https://golang.org/doc/install#tarball).
-   * Make sure that your GOPATH and PATH are configured correctly
+1. Download and install [Go 1.16 or later](https://golang.org/doc/install#tarball).
+   - Make sure that your `GOPATH` and `PATH` environment variables are configured so that gotest modules can be run. For example:
+
    ```bash
    export GOPATH=~/go
    export PATH=$PATH:$GOPATH/bin
    ```
-2. [Delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) for Debugging
+
+2. Install the [Delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) debugger for Go.
+
+3. Install [golangci-lint](https://golangci-lint.run/usage/install).
 
 ### Windows
 
 1. The Go language environment [(instructions)](https://golang.org/doc/install#windows).
```

Comment:
keep same format with Linux:
```Download and install [Go 1.16 or later]```

Reply:
I fixed up the wording to be more consistent with the Linux side, but I wouldn't get too hung up on this for now; as suggested in #2529, the Windows instructions are going to be reworked more in an upcoming PR anyway.